src/qt/addresstablemodel.cpp

```diff
@@ -143,8 +143,12 @@ class AddressTablePriv
             var = &coldSendNum;
         } else if (purpose == AddressBook::AddressBookPurpose::DELEGABLE || purpose == AddressBook::AddressBookPurpose::DELEGATOR) {
             var = &dellNum;
+        } else {
+            return;
+        }
+        if (var != nullptr) {
```

Comment:
Minor styling nit, initialize `var` in `nullptr` and here only do `if (var) {`.

Reply:
added nullptr initialization but left this condition as is (same as `if(var)` but more readable).
Shouldn't be even needed given the `else return`, but still better to include it, imo.